tests/ixia/ecn/files/helper.py

```diff
@@ -40,10 +43,15 @@ def run_ecn_test(api,
                  kmax,
                  pmax,
                  pkt_size,
-                 pkt_cnt,
                  lossless_prio,
                  prio_dscp_map,
-                 iters):
+                 iters,
+                 pkt_cnt=NUMBER_OF_TEST_PACKETS,
```

Comment:
Can we rename this to data_pkt_cnt to differentiate with # of PFC packets?

Reply:
Done.